AcademicReward/AcademicReward.csproj

```diff
@@ -53,13 +53,21 @@
 		<MauiAsset Include="Resources\Raw\**" LogicalName="%(RecursiveDir)%(Filename)%(Extension)" />
 	</ItemGroup>
 
+	<ItemGroup>
```

Comment:
Do you know what this remove is doing here? Is this adding the images to your list? I'm curious